### PR TITLE
First draft of sign up form

### PR DESCRIPTION
**Pull request by HYFar**
Will need some help on the server side

### CODE REVIEW

**Review by prabhamanju — COMMENTED** [review]
good efforts  after my review… some points need to implement….
1)login element should not be there in navbar.
2)This form should matched with the user model as user is also a mentor. And the details will be saved in that model….(plz check with Michiel and Anthony)
3)there should be checkbox for both mentor and mentees as only one singup page is used for registration.(check box as user wants to register as mentor or as mentees) 
4)When user enters the details that should be display in placeholder. And when user click signup button placeholder should reset to original.

**Comment by archived-m** [comment]
@prabhamanju quick remark on your third comment:

Users should be able to set their mentor/mentee status even after having registered (from the profile maybe?). Otherwise you'd have to register a new account if I'd like to be a mentor if I was previously a mentee.

File: client/src/components/SignUpPageBody.jsx [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+      [e.target.name]: e.target.value
+    });
+    console.log(this.state);
+  }
+
+  handleSubmit = event => {
+    event.preventDefault();
+    const { email, password, firstName, lastName, age, zipCode } = this.state;
+
+    fetch(`http://localhost:4000/user`, {
```
Comment (Toinne) [inline client/src/components/SignUpPageBody.jsx:31]: Try and extract api communication to the `api.js` file

File: client/src/components/SignUpPageBody.jsx [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+      zipCode: ""
+    };
+    this.handleInputChange = this.handleInputChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleInputChange(e) {
+    console.log(e.target);
+    this.setState({
+      [e.target.name]: e.target.value
```
Comment (Toinne) [inline client/src/components/SignUpPageBody.jsx:22]: Cool trick

File: client/src/App.jsx [hunk 3]
```diff
@@ -4,6 +4,8 @@ import Home from "./components/Home";
 import UserProfile from "./components/UserProfile";
 import Mentors from "./components/Mentors";
 import Contact from "./components/ContactPage";
+import SignUp from "./components/SignUpPage";
```
Comment (Toinne) [inline client/src/App.jsx:7]: Imports are a bit confusing here you import the component `Contact ` as `SignUp` from the file `SignUpPage`

**Comment by prabhamanju** [comment]
I created a new registration page for user so closing this issue.

**Pull request closed without merging**